packages/core/src/components/DropdownNew/hooks/useDropdownMultiCombobox.ts

```diff
@@ -0,0 +1,93 @@
+import { useCombobox } from "downshift";
+
+import { useMemo } from "react";
+import useDropdownFiltering from "./useDropdownFiltering";
+import { useMultipleSelection } from "downshift";
```

Comment:
```suggestion
import { useMemo } from "react";
import useDropdownFiltering from "./useDropdownFiltering";
import { useCombobox, useMultipleSelection } from "downshift";
```

not critical of course, lol 😅